src/components/NavBar/NavBarPresentation.js

```diff
@@ -49,9 +51,52 @@ function NavBarPresentation({
                   <span className="sr-only">FirstVoices Logo</span>
                   <Link to="/">{fvlogo}</Link>
                 </div>
-                <div className="w-16 text-white text-sm leading-tight text-center mr-2">
+                <button
+                  className="w-16 text-white text-sm leading-tight text-center mr-2"
+                  onClick={() => setBetaModalOpen(true)}
+                  type="button"
+                >
                   Beta Version
-                </div>
+                </button>
+
+                <Modal.Presentation
+                  isOpen={betaModalOpen}
+                  closeHandler={() => setBetaModalOpen(false)}
+                >
+                  <div
+                    data-testid="BetaModalContent"
+                    className="bg-white rounded-lg shadow-lg px-6 py-32"
+                  >
+                    <h2 className="text-3xl font-bold tracking-tight text-fv-charcoal sm:text-4xl">
+                      Welcome to the new FirstVoices!
+                    </h2>
+                    <p className="mx-auto mt-3 md:mt-6 max-w-xl text-lg leading-8 text-fv-charcoal-light">
+                      The beta version of FirstVoices v3 is now live.
+                      <br /> If you are looking for the old version of
+                      FirstVoices, <br />
+                      please click{' '}
+                      <a
+                        href="https://www.archive.firstvoices.com"
```

Comment:
Minor: We may want this link to open in a new tab when clicked.

Reply:
good catch